modules/nw-udn-additional-config-details.adoc

```diff
@@ -0,0 +1,31 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-udn-additional-config-details_{context}"]
+== Additional configuration details for a UserDefinedNetworks CR
+
+The following table explains additional configurations for UDN that are optional. It is not recommended to set these fields without explicit need and understanding of OVN-Kubernetes network topology.
+
+.`UserDefinedNetworks` optional configurations
+[cols="1,1,7", options="header"]
+|====
+
+|Field|Type|Description
+
+|`spec.joinSubnets`
+|object
+|When omitted, the platform sets default values for the `joinSubnets` field of `100.65.0.0/16` for IPv4 and  `fd99::/64` for IPv6. If the default address values are used anywhere in the cluster's network you must override it by setting the `joinSubnets` field. If you choose to set this field, ensure it does not conflict with other subnets in the cluster such as the cluster subnet, the `default` network cluster subnet, and the masquerade subnet.
+
+The `joinSubnets` field configures the routing between different segments within Layer 2 and Layer 3 networks. Dual-stack clusters can set 2 subnets, one for each IP family; otherwise, only 1 subnet is allowed. This field is only allowed for the `Primary` network.
```

Comment:
```suggestion
The `joinSubnets` field configures the routing between different segments within a UDN network. Dual-stack clusters can set 2 subnets, one for each IP family; otherwise, only 1 subnet is allowed. This field is only allowed for the `Primary` network.
```